### PR TITLE
Update docker-compose.*.yml

### PR DESCRIPTION
**Pull request by weidazhao**

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: '2'
+
+services:
+  marketing:
+    build:
+      context: ./src/Marketing
+      dockerfile: Dockerfile
```
Comment (stepro) [inline docker-compose.override.yml]: This line not strictly necessary. Is it included just for informative purposes?

File: src/Marketing/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,6 @@
 FROM microsoft/aspnetcore:1.0.1
-ARG source=bin/Release/publishoutput
+ARG source=bin/Release/PublishOutput
```
Comment (stepro) [inline src/Marketing/Dockerfile]: The name of this argument is confusing. Shouldn't it be something like "binaries" or "buildOutput"?

Comment (stepro) [inline src/Marketing/Dockerfile]: What happens if DOCKER_BUILD_SOURCE is not set? Does it pick the default value?

Comment (weidazhao) [inline src/Marketing/Dockerfile]: bin/Release/PublishOutput is the default of dotnet publish. if DOCKER_BUILD_SOURCE is not set, Dockerfile will use the default, bin/Release/PublishOutput.

File: docker-compose.override.yml (hunk 1 above)
Comment (weidazhao) [inline docker-compose.override.yml]: Which line exactly were you referring to?

**Comment by stepro** [comment]
David, can we get these changes checked in to the repo? Then I'll update this repo with the CI compose files.

**Pull request merged**